tutorials/03_multi_criteria_decision_analysis.tex

```diff
@@ -135,7 +128,7 @@ \subsection*{Preferences: matching elicitation}
 You can also determine your preferences by comparing alternative outcome scenarios, in the Matching elicitation process.
 \newline
 
-\noindent \leftpointright \, Click the ‘Matching’ button under the weights table on the preferences tab. Select ‘ARR’ as the most important criterion in Step 1. In step 2, click the blue question mark in the row for ‘Hep. AE.’ In the dialog that appears (Figure \ref{fig:matching1}), you are presented with a graph and two alternative scenarios. The graph is the indifference curve. You can change the value for ARR in Alternative B by clicking on it and moving the slider. This will update the graph. Change the value until you feel that the two alternatives are approximately equal, ie. decreasing ARR by a certain amount is ‘worth’ about that much increase in Hep. AE. Then click ‘Save’. The table will now have updated with the corresponding weight (Figure \ref{fig:matching2}).
+\noindent \leftpointright \, Click the ‘Matching’ button under the weights table on the preferences tab. Select ‘ARR’ as the most important criterion in Step 1. In step 2, you can change the value for ARR in Alternative B by moving the slider. Change the value until you feel that the two alternatives are approximately equal, ie. decreasing ARR by a certain amount is ‘worth’ about that much increase in Hep. AE. Click 'Next' and complete the elicitation for each remainding criterion.
```

Comment:
'remainding'

Reply:
it's a criterion which remains and reminds